_posts/2015-11-24-booting-discovery-over-http.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: 
+date: 2015-11-24 11:14:15
+author: Foreman
+tags:
```

Comment:
> Oh, this isn't a repost from lukas.zapletovi.com, is it? Never mind then :)

Oh ok, anyway what tags do you mean and how do I set it? I cannot find
any example on our site. I want to repost something else.

-- 
Later,
 Lukas #lzap Zapletal